meson.build

```diff
@@ -42,7 +42,7 @@ is_windows = host_machine.system() == 'windows'
 is_x64 = host_machine.cpu_family() == 'x86_64'
 is_subproject = meson.is_subproject()
 has_exceptions = get_option('cpp_eh') != 'none'
-include_dirs = include_directories('include', 'external')
+include_dirs = include_directories('include', 'subprojects')
```

Comment:
Please exclusively use `dependency()`, meson gets mad at you if you try to reference `subprojects/` as a path.

Due to an implementation bug, it didn't warn you if you literally used `subprojects/`, which I'm fixing in https://github.com/mesonbuild/meson/pull/9997